platforms/core-runtime/java-language-extensions/src/main/java/org/gradle/api/JavaVersion.java

```diff
@@ -208,13 +208,12 @@ public static JavaVersion current() {
         return currentJavaVersion;
     }
 
-    @VisibleForTesting
     static void resetCurrent() {
```

Comment:
❓ Why is that not `VisibleForTesting` anymore?

Reply:
This project needs to be Java 6 compatible, which means no Guava dependency. We can't mark it as `@VisibleForTesting`. Note that this is a non-public method, which we typically only use in the codebase to expose things to tests, so it's already indicative.